src/main/osd/osd_elements.c

```diff
@@ -475,7 +475,7 @@ static void osdElementAltitude(osdElementParms_t *element)
         osdFormatAltitudeString(element->buff, getEstimatedAltitudeCm());
     } else {
         element->buff[0] = SYM_ALTITUDE;
-        element->buff[1] = SYM_COLON; // We use this symbol when we don't have a valid measure
+        element->buff[0] = SYM_HYPHEN; // We use this symbol when we don't have a valid measure
```

Comment:
I think this is where the unit test failed.

Reply:
Good catch! It must be buff[1] and not buff[0]

Reply:
Yeah, looks like I stuffed up the rebase. Good thing we have tests. Fixed now.